moda-contracts/test/Management.t.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import {Test, console2} from "forge-std/Test.sol";
+import "../src/Management.sol";
+
+contract ManagementTest is Test {
+    Management public management;
+
+    address public artist = address(0x1);
+    address[] managers;
+    address managerOne = address(0x3);
+    address managerTwo = address(0x4);
+
+    function setUp() public {
+        management = new Management();
+        managers = new address[](2);
+        managers[0] = managerOne;
+        managers[1] = managerTwo;
+    }
+
+    function addManagers_setUp() public {
+        vm.startPrank(artist);
+        management.addManagers(managers);
+        vm.stopPrank();
+    }
+
+    function test_addManagers() public {
+        addManagers_setUp();
+        bool isManagerOne = management.isManager(artist, managerOne);
+        bool isManagerTwo = management.isManager(artist, managerTwo);
+        uint256 managerCount = management.getManagerCount(artist);
+        address managerOneAddress = management.getManager(artist, 0);
+        address managerTwoAddress = management.getManager(artist, 1);
+
+        assertTrue(isManagerOne);
+        assertTrue(isManagerTwo);
+        assertEq(managerCount, 2);
+        assertEq(managerOneAddress, managerOne);
+        assertEq(managerTwoAddress, managerTwo);
+    }
+
+    function test_removeManagers() public {
+        addManagers_setUp();
+        vm.startPrank(artist);
+        management.removeManagers(managers);
+        vm.stopPrank();
+        bool isManagerOne = management.isManager(artist, managerOne);
+        bool isManagerTwo = management.isManager(artist, managerTwo);
+        uint256 managerCount = management.getManagerCount(artist);
+
+        assertFalse(isManagerOne);
+        assertFalse(isManagerTwo);
+        assertEq(managerCount, 0);
+    }
+
+    // reverts
```

Comment:
I would break the tests up into sections for each contract method vs "successful functions" and "reverts". 

e.g. `// addManagers` 

1. Then you test it adds managers
2. Test it reverts under some condition (as many tests as needed)
3. Then one that it emits an event with expected values. 

This will force you to think through each scenario per method. Making them granular like this is good bc/ when something fails we know exactly what it is